### PR TITLE
Added an "npc" actor type and added a compendium of NPCs.

### PR DESCRIPTION
**Pull request by justinross**
I'm hoping I did this correctly (it seems to work on my end). For Issue #38 

I created an npc Actor type so that these NPCs can be used for other purposes in the system. I created an `npcs` compendium and added all of the npcs from the character sheets and crew sheets (the npc Actor type has fields for "associated_crew_type" and "associated_class"), as well as a handful that were listed as "Doskvol Notables" in the sheets PDF.

There were a couple of NPCs that showed up twice, so I added them twice (Nyryx, for example, is an option on both the Spider and Whisper playbooks, but with different descriptions. On the Spider's sheet, she's a prostitute. On the Whisper sheet, she's a ghost. In Doskvol Notables, she's a rogue spirit inhabiting a prostitute. Not sure how to handle that). Laroze, on the other hand, is just described as "A Bluecoat" on both of their playbook appearances. Situations like that could get handled by keeping two entries in the compendium, or by making `associated_class` (and maybe `associated_crew_type`) arrays instead of just text fields, I guess?

(Oh, foundry also seems to have deleted some duplicate entries in the factions.db pack while I was working on this, so I left that change in)

/edit: And apparently while trying to merge the NPC changes into my redesign branch, I did it the other way around and now they're here. Should be resolved now.

### CODE REVIEW

File: packs/factions.db [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
...row’s Foot as their turf. Everyone in the district pays up the chain to them. HQ in an abandoned City Watch tower. Operates gambling dens in Crow’s Foot and extortion rackets at the Docks.","assets":"A veteran gang of thugs and killers. A number of small boats. A fortified HQ.","quirks":"Roric’s body was lost during his murder (it fell into a canal). His vengeful ghost is now at large in the city.","notables":"Lyssa (leader, brash, killer, noble family). Bell (second-in-command, loyal).","allies":"The Bluecoats, Sailors, The Lost, Citizens of Crow’s Foot.","enemies":"The Hive, Inspectors, Dockers.","situation":"Lyssa murdered the former boss of the Crows, Roric. She is a fearsome killer, and few want to cross her, but her position as leader of the Crows is uncertain. Some were very loyal to Roric. As the power-play continues, the Crows’ hold on the district just might slip away.","goal_clock":0,"hold":{"value":1,"max":2,"max_default":2,"name_default":"BITD.Hold","name":"BITD.Hold"},"status":{"value":[4],"max":7,"max_default":7,"name_default":"BITD.Status","name":"BITD.Status"}},"flags":{},"img":"systems/blades-in-the-dark/styles/assets/icons/Icon.4_27.png","notes":"","effects":[]}u
```
Comment (megastruktur) [inline packs/factions.db:6]: Looks like there is a typo. Letter "u" is added to the end of that string.

File: template.json [hunk 2]
```diff
@@ -1,32 +1,32 @@
 {
-"Actor": {
-  "types": ["character", "crew", "\uD83D\uDD5B clock", "factions"],
-  "character": {
-    "alias": "",
-    "stress": {
+  "Actor": {
```
Comment (megastruktur) [inline template.json:2]: Please make sure to maintain the same amount of tabs

File: packs/factions.db (hunk 1 above)
Comment (justinross) [reply packs/factions.db:6]: Ah, weird. I can address tomorrow.

File: packs/factions.db [hunk 3]
```diff
@@ -40,8 +40,4 @@
...
...ark-craft technicians, demonologist Whispers, and void-touched navigators. Companies of marines to protect the vessels and their valuable cargo at sea and in port.","quirks":"","notables":"Lord Strangford (captain, ruthless, arrogant, tainted). Lady Clave (captain, daring, cruel, accomplished). Lady Ankhayat (Iruvian captain, confident, charming, scoundrel).","allies":"City Council, The Church of Ecstasy, Sailors, Dockers, Sparkwrights.","enemies":"The Grinders, Ministry of Preservation, The Path of Echoes.","situation":"The captains have a horrible secret: the known hunting grounds for leviathans are coming up barren. The immortal creatures, once so reliable in their movements in the Void Sea, have begun to migrate elsewhere. New hunting grounds must be found before the surplus of leviathan blood disappears, and with it, the lightning barriers and the survival of the human race.","goal_clock":0,"hold":{"value":2,"max":2,"max_default":2,"name_default":"BITD.Hold","name":"BITD.Hold"},"status":{"value":[4],"max":7,"max_default":7,"name_default":"BITD.Status","name":"BITD.Status"}},"flags":{},"img":"systems/blades-in-the-dark/styles/assets/icons/Icons8_16.png","notes":"","effects":[]}
```
Comment (megastruktur) [inline packs/factions.db:43]: Please make sure to have a newline in the end of the file  :)

**Review by megastruktur — CHANGES_REQUESTED** [review]
Hey bro, can you please fix the things? Then I'll merge and make a new 2.17 version.

File: packs/factions.db (hunk 3 above)
Comment (justinross) [reply packs/factions.db]: Added.

File: template.json (hunk 2 above)
Comment (justinross) [reply template.json]: Should be fixed.

File: packs/factions.db (hunk 1 above)
Comment (justinross) [reply packs/factions.db]: Removed.

File: packs/factions.db [hunk 4]
```diff
@@ -41,7 +41,3 @@
...
...Crow’s Foot as their turf. Everyone in the district pays up the chain to them. HQ in an abandoned City Watch tower. Operates gambling dens in Crow’s Foot and extortion rackets at the Docks.","assets":"A veteran gang of thugs and killers. A number of small boats. A fortified HQ.","quirks":"Roric’s body was lost during his murder (it fell into a canal). His vengeful ghost is now at large in the city.","notables":"Lyssa (leader, brash, killer, noble family). Bell (second-in-command, loyal).","allies":"The Bluecoats, Sailors, The Lost, Citizens of Crow’s Foot.","enemies":"The Hive, Inspectors, Dockers.","situation":"Lyssa murdered the former boss of the Crows, Roric. She is a fearsome killer, and few want to cross her, but her position as leader of the Crows is uncertain. Some were very loyal to Roric. As the power-play continues, the Crows’ hold on the district just might slip away.","goal_clock":0,"hold":{"value":1,"max":2,"max_default":2,"name_default":"BITD.Hold","name":"BITD.Hold"},"status":{"value":[4],"max":7,"max_default":7,"name_default":"BITD.Status","name":"BITD.Status"}},"flags":{},"img":"systems/blades-in-the-dark/styles/assets/icons/Icon.4_27.png","notes":"","effects":[]}
```
Comment (megastruktur) [inline packs/factions.db:45]: Hey, why did you remove The Crows, The Horde and Citizenry from Factions.db?  :)  Is it by a mistake or intentionally?

**Review by megastruktur — CHANGES_REQUESTED** [review]

File: packs/factions.db (hunk 4 above)
Comment (justinross) [reply packs/factions.db:45]: There were some duplicates that I removed. I’m seeing those still in there, on lines 1, 6, and 25. Am I confused and looking at it wrong?

**Review by megastruktur — APPROVED** [review]

File: packs/factions.db (hunk 4 above)
Comment (megastruktur) [reply packs/factions.db:45]: Nah it's fine, I've missed these. Ok, I will create a new release then  :)
